backend/de.metas.shipper.gateway.dhl/src/main/java/de/metas/shipper/gateway/dhl/process/DhlLabel_Download.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * #%L
+ * de.metas.shipper.gateway.dhl
+ * %%
+ * Copyright (C) 2022 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.shipper.gateway.dhl.process;
+
+import de.metas.process.IProcessPrecondition;
+import de.metas.process.IProcessPreconditionsContext;
+import de.metas.process.JavaProcess;
+import de.metas.process.ProcessPreconditionsResolution;
+import de.metas.shipper.gateway.dhl.model.DhlCustomDeliveryDataDetail;
+import de.metas.shipper.gateway.dhl.model.DhlShipmentOrderId;
+import de.metas.shipper.gateway.dhl.model.IDhlShipmentOrderDAO;
+import de.metas.util.Services;
+import lombok.NonNull;
+import org.compiere.util.MimeType;
+import org.springframework.core.io.ByteArrayResource;
+
+public class DhlLabel_Download extends JavaProcess implements IProcessPrecondition
```

Comment:
Please rename to `DHL_ShipmentOrder_...`, e.g. `DHL_ShipmentOrder_DhlLabel_Download`
The process vaue and classname should start with the corresponding table